pkg/cli/cmd/credential/register/azure/sp/serviceprincipal_test.go

```diff
@@ -104,15 +101,6 @@ func Test_Validate(t *testing.T) {
 			ExpectedValid: false,
 			ConfigHolder:  framework.ConfigHolder{Config: configWithWorkspace},
 		},
-		{
-			Name: "Azure command without subscription",
-			Input: []string{
-				"--client-id", "abcd",
-				"--client-secret", "efgh",
-			},
-			ExpectedValid: false,
-			ConfigHolder:  framework.ConfigHolder{Config: configWithWorkspace},
-		},
```

Comment:
Looks like a bad copy-paste? and this was redundant with L99?

Reply:
I think so